<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P3DViewer</class>
 <widget class="QWidget" name="P3DViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QListWidget" name="objects"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="objectsLbl">
     <property name="text">
      <string>Objects</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="images"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="imagesLbl">
     <property name="text">
      <string>Images</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QListWidget" name="markers"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="markersLbl">
     <property name="text">
      <string>Markers</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="objCount">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="imgCount">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="mrkCount">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
